pages/1.13/security/oss/authentication/troubleshooting/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: layout.pug
+navigationTitle: Troubleshooting
+title: Troubleshooting Authentication 
+excerpt: Troubleshooting authentication issues in DC/OS
+
+menuWeight: 50
+---
+# Ad Blockers and the DC/OS UI
+
+During testing, we have observed issues with loading the DC/OS UI login page when certain ad blockers such as HTTP Switchboard or Privacy Badger are active. Other ad blockers like uBlock Origin are known to work.
```

Comment:
Rewrite line #11 (Use the following sentences if it aligns to implementation work).
During testing, there was no issue involved with loading the DC/OS UI login page using ad blockers like `uBlock Origin`. But certain ad blockers such as `HTTP Switchboard` or `Privacy Badger` caused issues during active state.